configurations/tablets.yaml

```diff
@@ -1,3 +1,5 @@
+nemesis_class_name: 'SisyphusMonkey'
```

Comment:
I'm not sure that you want to introduce those here.

this configuration file is used in performance tests, and overwriting those would break them

Reply:
...if it's only meant to be used in performance, then it really shouldn't have been named just tablets.yaml. That's confusing.
But shouldn't we use the same nemesis class for tablets performance tests as well? Since those are the only ones that are supported by tablets in the first place? @bhalevy @roydahan @xemul

Reply:
@xemul is using it in a case we it's selecting a very specific nemesis, and this change would override it.

Reply:
Moved my config to `configurations/tablets_supported_nemeses.yaml`, but I still use `configurations/tablets.yaml, would that be alright?